admin/includes/init_includes/init_admin_auth.php

```diff
@@ -10,6 +10,22 @@
 
 define('SUPERUSER_PROFILE', 1);
 
+// -----
+// Special handling for AJAX requests.  Return a 'logged_out' error if no admin session
+// is active; otherwise, bypass the remainder of the authorization checks.
+//
+if (basename($PHP_SELF) != FILENAME_AJAX . '.php') {

```

Comment:
Should this be `==` instead of `!=` ?

Reply:
It's not my month ... of course it should be.